<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppliedCrytographyClass</class>
 <widget class="QMainWindow" name="AppliedCrytographyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Applied Cryptography</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="MainTabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>840</width>
      <height>540</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Encryption">
     <attribute name="title">
      <string>Encryption</string>
     </attribute>
     <widget class="QGroupBox" name="Encryption_Public_Key_Group">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>371</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Public Key</string>
      </property>
      <widget class="QLabel" name="E_Value_N">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Value N:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Encryption_N">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>331</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="E_Value_B">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Value B:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Encryption_B">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>331</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Save_Public_Key">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>158</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Public Key</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="Encryption_Level">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>12</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Encryption Level:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="RSA_Level">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>RSA-512(not safe)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RSA-768(not safe)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RSA-1024(safe)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RSA-2048(safe)</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Generate_Key">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>9</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Key</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="Encryption_Private_Key_Group">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>371</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Private Key</string>
      </property>
      <widget class="QLabel" name="E_Value_A">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Value A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="E_Value_P">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Value P:</string>
       </property>
      </widget>
      <widget class="QLabel" name="E_Value_Q">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Value Q:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Encryption_P">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>331</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Encryption_A">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>331</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Encryption_Q">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>331</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Save_Private_Key">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>228</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Private Key</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Encryption_Message_Group">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>431</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Message</string>
      </property>
      <widget class="QTextEdit" name="Encrption_Message">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>391</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Load_Message">
       <property name="geometry">
        <rect>
         <x>311</x>
         <y>190</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Message</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Encryption_Encrypted_Message_Group">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>240</y>
        <width>431</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Encrypted Message</string>
      </property>
      <widget class="QPushButton" name="Encrypt_Message">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Encrypt Message</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Save_Message">
       <property name="geometry">
        <rect>
         <x>311</x>
         <y>20</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Message</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Encryption_Encrypted_Message">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>53</y>
         <width>391</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Decryption">
     <attribute name="title">
      <string>Decryption</string>
     </attribute>
     <widget class="QGroupBox" name="Decryption_Key_Group">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>811</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Keys</string>
      </property>
      <widget class="QPushButton" name="Load_Private_Key">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>20</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Private Key</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Load_Public_Key">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>100</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Public Key</string>
       </property>
      </widget>
      <widget class="QLabel" name="D_Value_A">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>28</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Value A:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Decryption_A">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>771</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="D_Value_N">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>108</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Value N:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Decryption_N">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>771</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Decryption_Encrypted_Text_Group">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>401</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Encrypted Text</string>
      </property>
      <widget class="QPushButton" name="Load_Encrypted_Text">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>25</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Encrypted Text</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Decryption_Encrypted_Text">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>361</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Decryption_Decrypted_Text_Group">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>401</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Decrypted Text</string>
      </property>
      <widget class="QPushButton" name="Decrypt_Message">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>25</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Decrypt Message</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Save_Decrypted_Text">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>25</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Decrypted Text</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Decryption_Decrypted_Text">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>361</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="appliedcrytography.qrc"/>
 </resources>
 <connections/>
</ui>
